bin/modbus

```diff
@@ -45,17 +77,33 @@ def main():
     parser.add_argument('-p', '--stop-bits', type=int, default=1)
     parser.add_argument('-P', '--parity', choices=['e', 'o', 'n'], default='n')
     parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument('-t', '--timeout', type=float, default=5.0)
     parser.add_argument('device')
     parser.add_argument('access', nargs='+')
     args = parser.parse_args()
 
-    if args.verbose:
-        logging.basicConfig(format='\033[90m%(message)s\033[39m', level=logging.DEBUG)
+    clr.init()
+    try:
+
+        mainLogger = logging.getLogger()            # Main logger
```

Comment:
Here too, pep8

Reply:
This is a case where I agree with pep8. Also: whoops.